cli/commit.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/reference"
+
+	"github.com/spf13/cobra"
+)
+
+// commitDescription is used to describe commit command in detail and auto generate command doc.
+var commitDescription = "commit an image from a container."
+
+// CommitCommand is used to implement 'commit' command.
+type CommitCommand struct {
+	baseCommand
+	author  string
+	message string
+}
+
+// Init initializes CommitCommand command.
+func (cc *CommitCommand) Init(c *Cli) {
+	cc.cli = c
+	cc.cmd = &cobra.Command{
+		Use:   "commit [OPTIONS] CONTAINER [REPOSITORY:TAG]",
```

Comment:
should be commit [OPTIONS] CONTAINER REPOSITORY[:TAG]

because `[REPOSITORY:TAG]` means option, not required